src/modules/CreateDrops/TicketDrop/AdditionalGiftsForm/AdditionalGiftsForm.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, TabPanel, TabPanels } from '@chakra-ui/react';
+import { useFormContext } from 'react-hook-form';
+import { useEffect } from 'react';
+
+import { RoundedTabs, TabListItem } from '@/common/components/RoundedTabs';
+import { LinkIcon, POAPNftIcon } from '@/common/components/Icons';
+
+import { CreateTicketFieldsSchema } from '../CreateTicketDropContext/CreateTicketDropContext';
+
+import { TokenForm } from './TokenForm';
+import { POAPNftForm } from './POAPNftForm';
+
+interface AdditionalGiftTabItem extends TabListItem {
+  name: 'token' | 'poapNft';
+}
+
+const tabList: AdditionalGiftTabItem[] = [
+  {
+    name: 'token',
+    label: 'Token',
+    icon: <LinkIcon h={{ base: '5' }} w={{ base: '5' }} />,
```

Comment:
```suggestion
    icon: <LinkIcon h='5'w='5' />,
```

Reply:
sorry forgot to add in the mobile font sizes